INSTALL.asciidoc

```diff
@@ -350,6 +356,12 @@ config:
 ~/path/to/qutebrowser/.venv/bin/python3 -m qutebrowser -c .qutebrowser-local "$@"
 ----
 
+To use the QtWebEngine backend on Windows, it is necessary to set the following environment variable to point to the QtWebEngineProcess.exe binary (adjust path as needed):
+
+----
+set QTWEBENGINEPROCESS_PATH=C:\Python35\Lib\site-packages\PyQt5\Qt\bin\QtWebEngineProcess.exe
+----
+
```

Comment:
I don't think we should mention this - it's still broken then anyways, and in the long run, we better fix this properly :wink: